frontend/app/components/wp-table/wp-table.directive.html

```diff
@@ -131,7 +131,8 @@
             </accessible-checkbox>
             <span>
               <a class="wp-table--details-link"
-                 ui-sref="work-packages.list.details.overview({workPackageId: row.object.id})">
+                 ui-state="desiredSplitViewState"
+                 ui-state-params="{workPackageId: row.object.id}">
```

Comment:
Ah, I wasn't aware that it might change. It that case, ok. The advantage of ui-sref is, that it creates a true href link. "open in new tab", etc. work as expected. However, if this is not required, a simple ng-click would do and the controller could initiate the state transition.